packages/memory/test/benchmark.ts

```diff
@@ -0,0 +1,985 @@
+/**
+ * Benchmarks for memory fact operations: set, get, retract
+ *
+ * Run with: deno bench test/benchmark.ts
```

Comment:
nit, but if you rename this to <something>.bench.ts then `deno bench` in the parent directory will run this and all other benchmarks.